### PR TITLE
Improve consistency between session and storage

### PR DESCRIPTION
**Pull request by leplatrem**
* [x] Put session PG schema in a sql file
* [x] Add documentation about session backends
* [x] Fix inclusion of storage backend documentaiton
* [x] Document loading of schema from SQL files directly

### CODE REVIEW

**Comment by leplatrem** [comment]
@ametaireau r?

(I know you're working on sessions with a potential different approach, but this applies to current code)

**Comment by Natim** [comment]
LGTM

File: docs/storage.rst [hunk 1]
```diff
@@ -3,5 +3,29 @@ Storage
...
+=====
+
+.. autoclass:: cliquet.storage.redis.Redis
+
+
+Memory
+======
+
+.. autoclass:: cliquet.storage.memory.Memory
+
```
Comment (leplatrem) [inline docs/storage.rst]: XXX: add cloud storage

**Comment by ametaireau** [comment]
Yep, looks good to me, let's merge!

**Pull request merged**